api_backend.yaml

```diff
@@ -221,6 +221,7 @@ paths:
         type: string
         required: true
         description: The ID of the message.
+      - $ref: "#/parameters/PublicMessage"
```

Comment:
We should modify response type too, to match new [GetMessageResonse](https://github.com/pagopa/io-functions-app-messages/blob/41a5c3e11b23b4b038e658bd951e9f9827e8304e/openapi/index.yaml#L1313)

Reply:
Sorry, you've done it.